pubspec.yaml

```diff
@@ -12,5 +12,10 @@ dependencies:
   sidekick_core: '>=0.7.1 <1.0.0'
 
 dev_dependencies:
-  lint: ^1.5.0
-  sidekick_plugin_installer: ^0.1.3
+  lint: ^2.0.0
+  sidekick_plugin_installer: ^0.2.1
+
+  # Template Dependencies
+  shelf_static: ^1.1.0
+  shelf_router: ^1.1.3
+  shelf_helmet: ^1.1.0
```

Comment:
@Giuspepe Interesting. I did not foresee this. Don't know if I like it.

Luckily those are dev_dependencies. Still, this might cause problems in the future.

Reply:
I thought about creating a template Pubspec in the template folder to manage the template dependencies